cpp/velox/substrait/SubstraitToVeloxExpr.cc

```diff
@@ -382,7 +393,8 @@ std::shared_ptr<const core::ConstantTypedExpr> SubstraitVeloxExprConverter::toVe
 ArrayVectorPtr SubstraitVeloxExprConverter::literalsToArrayVector(const ::substrait::Expression::Literal& literal) {
   auto childSize = literal.list().values().size();
   if (childSize == 0) {
-    return makeEmptyArrayVector(pool_);
+    // childSize == 0 should not happend here but just for integrity check
+    return makeEmptyArrayVector(pool_, UNKNOWN());
```

Comment:
Maybe it's better to throw for unexpected behavior.

Reply:
+1, if the child size is 0 that means it is a empty list, we should not go into this method. @WangGuangxin can you address this comment ?